helmfile.d/snippets/domains.gotmpl

```diff
@@ -6,15 +6,15 @@
 {{- $domainNames := list }}
 {{- $teams := set $tc.teams "admin" dict }}
 {{- range $teamId, $team := $tc.teams }}
-  {{- $baseDomain := printf "%s%s" (eq $teamId "admin" | ternary "" (printf "team-%s." $teamId)) $v.cluster.domain }}
+  {{- $baseDomain := printf "%s%s" (eq $teamId "admin" | ternary "" (printf "team-%s." $teamId)) $v.dns.domain }}
```

Comment:
I don't like to move that there...`cluster.domain` does not belong to the `dns:` block just because it exists in dns. The `dns` block is configuration for dns related tooling.

The `cluster.domain` is a prefix that goes with the cluster, and I would like to keep it there.

Reply:
I understand you point of view. Before reverting that change I would like to consult with you my reasoning. Are you okay with that?

Reply:
Well, I don't want to make this a big thing. Leave it if it has advantages and if you invested a lot in it

Reply:
ok